GetFed/GetFed/Models/Food.swift

```diff
@@ -26,9 +26,9 @@ struct FoodResults: Decodable {
 }
 
 struct Food: Decodable {
-    let label: String
-    let nutrients: Nutrients
-    let category: String
+    let label: String?
```

Comment:
Are you sure these are all optional in the API? It seems unlikely that every field may not exist.

Reply:
I've reverted these back in my latest pull request (commit 23938e8)